src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Charts/LineChart/SubscaleLineChart/ChartTooltip/ChartTooltip.test.tsx

```diff
@@ -36,31 +38,56 @@ describe('ChartTooltip', () => {
   test('renders component correctly when props data is null', () => {
     render(<ChartTooltip data={null} />);
     const tooltip = screen.queryByTestId(`${dataTestid}-tooltip`);
-    expect(tooltip).not.toBeInTheDocument();
+    expect(tooltip).toBeNull(); // Using Jest's default matcher to check if the element is not present
   });
 
   test('renders component correctly', () => {
     render(<ChartTooltip {...props} />);
     const tooltip = screen.getByTestId(`${dataTestid}-tooltip`);
```

Comment:
`getByTestId` will throw an error if the element is not present, rendering the subsequent null check unnecessary. We have to use the `query` version like the previous test

```suggestion
    const tooltip = screen.queryByTestId(`${dataTestid}-tooltip`);
```

Reply:
Does this only applies if we are testing for the element not to be null ? 
    expect(tooltip).not.toBeNull();
 Or what is the condition to use getByTestId instead of queryByTestId?

Reply:
They can be used interchangeably, they just produce different results so you have to handle them differently